app/src/production/google-services.json

```diff
@@ -0,0 +1,29 @@
+{
+  "project_info": {
+    "project_number": "946901486210",
+    "project_id": "dispathcer-prod",
+    "storage_bucket": "dispathcer-prod.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:946901486210:android:21186af00afbc6ee55429b",
+        "android_client_info": {
+          "package_name": "com.dispatcher.prod"
```

Comment:
as i was saying before, the convention of production env is that it's package_name is without '.prod'. 
fix it as well in the firebase project.